pkgs/development/tools/sshs/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "sshs";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "quantumsheep";
+    repo = pname;
+    rev = version;
+    sha256 = "D9doNVb2sTnzM8tF8cSJbIoaIYjGurkUHEyhcE3OqQg=";
+  };
+
+  vendorSha256 = "QWFz85bOrTnPGum5atccB5hKeATlZvDAt32by+DO/Fo=";
+
+  ldflags = [ "-w -s -X github.com/quantumsheep/sshs/cmd.Version=${version}" ];
```

Comment:
```suggestion
  ldflags = [ "-s" "-w" "-X github.com/quantumsheep/sshs/cmd.Version=${version}" ];
```

Reply:
apologies, resolved in [190a8c3](https://github.com/NixOS/nixpkgs/commit/190a8c326c5c7ce8bc8d28d4909a2f5e86d49fda)